opencti-platform/opencti-graphql/src/domain/task-common.js

```diff
@@ -67,6 +67,34 @@ export const createListTask = async (user, input) => {
   return listTask;
 };
 
+export const createListTask = async (user, input) => {
+  const { actions } = input;
+  checkActionValidity(user, actions);
+  const listTask = await buildListTask(user, input);
+  return listTask;
+};
+
+const isNotification = async (context, user, narrativeId) => {
```

Comment:
Do we really need to do this check ? 
If yes, it's better to use storeLoadByIds

Reply:
yes, to avoid background task launched from the API with this function for other entities than notifications and by a user that has not the right to do so.